cmd/cloud/workspace.go

```diff
@@ -219,9 +269,90 @@ func newWorkspaceTeamListCmd(out io.Writer) *cobra.Command {
 	return cmd
 }
 
-func listWorkspaceTeam(cmd *cobra.Command, out io.Writer) error {
-	cmd.SilenceUsage = true
-	return team.ListWorkspaceTeams(out, astroCoreClient, "")
+func newWorkspaceTokenCreateCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "create",
+		Aliases: []string{"cr"},
+		Short:   "Create a token in an Astro Workspace",
+		Long: "Create a token in an Astro Workspace\n$astro workspace token create --name [token name] --role [WORKSPACE_MEMBER, " +
+			"WORKSPACE_OPERATOR, WORKSPACE_OWNER].",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return createWorkspaceToken(cmd, out)
+		},
+	}
+	cmd.Flags().StringVarP(&tokenName, "name", "n", "", "The new token's name. If the name contains a space, specify the entire name within quotes \"\" ")
+	cmd.Flags().StringVarP(&tokenDescription, "description", "d", "", "Description of the token. If the description contains a space, specify the entire description in quotes \"\"")
+	cmd.Flags().StringVarP(&tokenRole, "role", "r", "", "The role for the "+
+		"token. Possible values are WORKSPACE_MEMBER, WORKSPACE_OPERATOR and WORKSPACE_OWNER")
+	cmd.Flags().IntVarP(&tokenExpiration, "expiration", "e", 0, "Expiration of the token in days. If the flag isn't used the token won't have an expiration. Must be between 1 and 3650 days. ")
+	return cmd
+}
+
+func newWorkspaceTokenUpdateCmd(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "update [TOKEN_ID]",
+		Aliases: []string{"up"},
+		Short:   "Update a token in an Astro Workspace",
```

Comment:
```suggestion
		Short:   "Update an Astro Workspace API token",
```
Can you also update an Organization API token that's assuming a Workspace role using this command? Might be worth including that detail in the long description.